edb/schema/links.py

```diff
@@ -374,11 +437,24 @@ class SetTargetDeletePolicy(sd.Command):
     astnode = qlast.OnTargetDelete
 
     @classmethod
-    def _cmd_from_ast(cls, schema, astnode, context):
-        return sd.AlterObjectProperty(property='on_target_delete')
+    def _cmd_from_ast(
```

Comment:
This class is dead code, remove it please.

Reply:
Do you mean the whole `SetTargetDeletePolicy`? If I remove this class, tests fail during arrangement phase.

Reply:
Ah, nevermind, it's not dead, I misread the code.